src/utils/bitConversion.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2015 - 2020, Nordic Semiconductor ASA
+ *
+ * All rights reserved.
+ *
+ * Use in source and binary forms, redistribution in binary form only, with
+ * or without modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions in binary form, except as embedded into a Nordic
+ *    Semiconductor ASA integrated circuit in a product or a software update for
+ *    such product, must reproduce the above copyright notice, this list of
+ *    conditions and the following disclaimer in the documentation and/or other
+ *    materials provided with the distribution.
+ *
+ * 2. Neither the name of Nordic Semiconductor ASA nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * 3. This software, with or without modification, must only be used with a Nordic
+ *    Semiconductor ASA integrated circuit.
+ *
+ * 4. Any software provided in binary form under this license must not be reverse
+ *    engineered, decompiled, modified and/or disassembled.
+ *
+ * THIS SOFTWARE IS PROVIDED BY NORDIC SEMICONDUCTOR ASA "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY, NONINFRINGEMENT, AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL NORDIC SEMICONDUCTOR ASA OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+ * TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+ * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/* eslint-disable no-bitwise */
+
+export const convertBits16 = b8 =>
+    (((b8 & 128) + 128) << 7) |
+    (((b8 & 64) + 64) << 6) |
+    (((b8 & 32) + 32) << 5) |
+    (((b8 & 16) + 16) << 4) |
+    (((b8 & 8) + 8) << 3) |
+    (((b8 & 4) + 4) << 2) |
+    (((b8 & 2) + 2) << 1) |
+    ((b8 & 1) + 1);
+
+export const doubleBitValue = (b16, n) => 3 & (b16 >> (n + n));
```

Comment:
I now have an idea what these two functions do but I probably will have a harder time in a year. So I suggest that you document them a bit and also write some tests for them.

You could probably use the examples from our chat about them and also what you wrote in the PR:
* 00: undefined
* 01: the channel was 0 during sampling
* 10: the channel was 1 during sampling
* 11: the channel had both 0 and 1 values
```js
bitTranslate(0b00000000).toString(2)
"0101010101010101"
bitTranslate(0b11111111).toString(2)
"1010101010101010"
(bitTranslate(0b00000000) | bitTranslate(0b11111111)).toString(2)
"1111111111111111"
```